setup/aio/010_Install_Puppet_Agent.rb

```diff
@@ -1,10 +1,23 @@
 test_name "Install Packages" do
 
   dev_builds_url  = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
+  nightly_builds_url = ENV['NIGHTLY_BUILDS_URL'] || 'http://nightlies.puppet.com'
+
+  sha = ENV['SHA']
+  release_stream = ENV['RELEASE_STREAM'] || 'puppet'
 
   step "Install puppet-agent..." do
-    sha = ENV['SHA']
-    install_from_build_data_url('puppet-agent', "#{dev_builds_url}/puppet-agent/#{sha}/artifacts/#{sha}.yaml", hosts)
+    # If SHA='latest', then we're installing from nightlies
+    if sha == 'latest'
```

Comment:
actually, I want to check both SHA and RELEASE_STREAM. If RELEASE_STREAM is set to something like, `puppet5-nightly` (i.e., ends with -nightly), then that should also trigger this block

Reply:
Nope, I changed my mind. That adds too much complication, especially for this round. I think this is fine as is.